Formula/rpm.rb

```diff
@@ -34,6 +34,13 @@ class Rpm < Formula
   depends_on "gettext"
   depends_on "xz"
   depends_on "ossp-uuid"
+  # use external libraries, until the internal have been fixed
+  depends_on "lua"
+  depends_on "syck"
+  depends_on "openssl"
```

Comment:
Have the issues with the bundled libraries been reported upstream and is switching to externally provided versions the solution recommended by upstream? I'd prefer if the underlying issue could be resolved instead of simply applying a quick workaround (at least that's what it looks like to me). Thanks!